### DISCUSSION TITLE
Preprocessing

### DISCUSSION THREAD

**Discussion by an3252sv** [AlexeyAB/darknet#8375] [Q&A]
Is it recommended to preprocess the dataset? For example by calculating the mean and standard deviation over the training set and then subtract the mean and divide with the standard deviation.

**Comment by Kjelldor** [comment]
It is recommended to preprocess as much of the data in your dataset you can, however, be careful not to remove any useful information.
Depending on the data you have and what you wish to detect within that data, your processing steps can become entirely different.

What you're talking about is normalizing the dataset to the standard Deviation. 
This can help, when detecting which parts of the data are considered outliers (e.g, which ones deviate a lot from the mean), but is detrimental if you are trying to detect something that is not dependant on the standarddeviation.

I recommend you ask yourself what it is within the data you wish to determine, and what information is needed to determine that. If you find information that is not useful in determining your outcome, try and remove it.
